.github/workflows/release_major.yml

```diff
@@ -0,0 +1,62 @@
+name: Release Major
+
+on:
+  workflow_dispatch:
+    inputs:
+      message:
+        required: true
+      add-commits:
+        type: boolean
+        default: true
+        description: Add commit messages to your release message.
+
+jobs:
+  Pre-Build:
+    uses: ./.github/workflows/build.yml
+    with:
+      godot-version: 3.3.2
+      ubuntu-version: 20.04
```

Comment:
I don't like that we have to build twice, but I also don't know if there is a better solution. Because the output will be changing based on the most recent tag (after we add that in) so we cant update the tag until after we get the hash from a build, and we cant build the release before updating the tag. So I think this is fine.

Reply:
There is no way around building it twice if we want to `compare the binary files` AND `add version information to the binaries during build`. I also dislike the idea, but after all we are wasting 2 minutes of a free public runner, that is not too bad.